spec/requests/organizations/expense_sheets_controller_spec.rb

```diff
@@ -0,0 +1,223 @@
+
+      before { sign_in organization_administrator.user }
+
+      it 'successfully fetches a list of expense sheets in the organization' do
+        perform_request
+        expect(response).to have_http_status(:success)
+        expect(response.body).to include 'Paul'
+        expect(response.body).not_to include 'Brigitte', 'Peter', 'Maria'
+      end
+
+      context 'when inactive services are turned on' do
+        let(:perform_request) { get organizations_expense_sheets_path(filters: {show_all: 'true'}) }
+
+        it 'fetches a list of all expense sheets in the organization' do
+          perform_request
+          expect(response.body).to include 'Paul', 'Maria'
+          expect(response.body).not_to include 'Brigitte', 'Peter'
+        end
+      end
+    end
+
+    context 'when a civil servant is signed in' do
+      let(:civil_servant) { create :civil_servant, :full }
+
+      before { sign_in civil_servant.user }
+
+      it_behaves_like 'unauthorized request' do
+        before { perform_request }
+      end
+    end
+
+    context 'when nobody is signed in' do
+      before { perform_request }
+
+      it_behaves_like 'unauthenticated request'
+    end
+  end
+
+  describe '#edit' do
+    let(:perform_request) { get edit_organizations_expense_sheet_path(paul_expense_sheet) }
+    let(:organization) { create :organization }
+
+    let(:brigitte) { create(:civil_servant, :with_service, :full, first_name: 'Brigitte') }
+    let(:maria) { create(:civil_servant, :with_service, :full, organization: organization, first_name: 'Maria') }
+    let(:paul) { create(:civil_servant, :with_service, :full, organization: organization, first_name: 'Paul') }
+
+    let(:brigitte_service) { brigitte.services.first }
+    let(:maria_service) { maria.services.first }
+    let(:paul_service) { paul.services.first }
+
+    let(:brigitte_expense_sheet) do
+      create(:expense_sheet, :locked, service: brigitte_service,
+             beginning: brigitte_service.beginning,
+             ending: brigitte_service.ending)
+    end
+    let(:maria_expense_sheet) do
+      create(:expense_sheet, :closed, service: maria_service,
+             beginning: maria_service.beginning,
+             ending: maria_service.ending)
+    end
+    let(:paul_expense_sheet) do
+      create(:expense_sheet, service: paul_service,
+             beginning: paul_service.beginning,
+             ending: paul_service.ending)
+    end
+
+    context 'when a organization administrator is signed in' do
+      let(:organization_administrator) { create :organization_member, organization: organization }
+
+      before { sign_in organization_administrator.user }
+
+      it 'successfully displays a expense sheet form in the organization' do
+        perform_request
+        expect(response).to have_http_status(:success)
+        expect(response.body).to include 'Paul'
+        expect(response.body).not_to include 'Maria', 'Brigitte'
```

Comment:
Why do you test that they are not included? I mean, if you request only paul then maria and brigitte won't be included anyway in edit

Why not test for the expense sheet values?